app/mailers/patient_mailer.rb

```diff
@@ -27,19 +27,19 @@ def enrollment_sms_weblink(patient)
     url_contents = new_patient_assessment_jurisdiction_report_lang_url(patient.submission_token,
                                                                        lang&.to_s || 'en',
                                                                        patient.jurisdiction.unique_identifier[0, 32])
-    account_sid = ENV['TWILLIO_API_ACCOUNT']
-    auth_token = ENV['TWILLIO_API_KEY']
-    from = ENV['TWILLIO_SENDING_NUMBER']
+    account_sid = ENV['TWILIO_API_ACCOUNT']
+    auth_token = ENV['TWILIO_API_KEY']
+    messaging_service_sid = ENV['TWILIO_MESSAGING_SERVICE_SID']
     client = Twilio::REST::Client.new(account_sid, auth_token)
     client.messages.create(
-      from: from,
       to: Phonelib.parse(patient.primary_telephone, 'US').full_e164,
-      body: intro_contents
+      body: intro_contents,
+      messaging_service_sid: messaging_service_sid
```

Comment:
With my above comment, perhaps this is an opportunity to DRY out all these calls to Twilio. Would also make it much easier to swap to another provider if someone wanted to. Maybe move it to a separate class, with a "provider" interface (has `send_sms`, `send_phone_call`, etc. - and we can drop in the actually mechanisms to make it work based on provider). Thoughts?

Reply:
Agreed, I think that's a great idea. Will do.